src/cascadia/LocalTests_SettingsModel/ColorSchemeTests.cpp

```diff
@@ -81,7 +81,6 @@ namespace SettingsModelLocalTests
         std::array<COLORREF, COLOR_TABLE_SIZE> expectedCampbellTable;
         const auto campbellSpan = gsl::make_span(expectedCampbellTable);
         Utils::InitializeColorTable(campbellSpan);
-        Utils::SetColorTableAlpha(campbellSpan, 0);

```

Comment:
This shouldn't be necessary, because the `COLORREF`s are automatically initialized with zero alpha when cast from `til::color` (which is what happens in the `InitializeColorTable` method).